tests/baselines/reference/docker/test262.log

```diff
@@ -0,0 +1,2913 @@
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-null.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-null.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-single-args.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-single-args.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-spread-operator.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-spread-operator.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-undefined.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/meth-args-trailing-comma-undefined.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/arguments-object/non-strict-arguments-object-is-immutable.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+PASS test/language/asi/S7.9.2_A1_T1.js
+PASS test/language/asi/S7.9.2_A1_T1.js
+FAIL test/language/asi/S7.9.2_A1_T2.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9.2_A1_T2.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+PASS test/language/asi/S7.9.2_A1_T3.js
+PASS test/language/asi/S7.9.2_A1_T3.js
+FAIL test/language/asi/S7.9.2_A1_T4.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9.2_A1_T4.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9.2_A1_T5.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9.2_A1_T5.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+PASS test/language/asi/S7.9.2_A1_T6.js
+PASS test/language/asi/S7.9.2_A1_T6.js
+FAIL test/language/asi/S7.9.2_A1_T7.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9.2_A1_T7.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9_A1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9_A1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9_A10_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9_A10_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/language/asi/S7.9_A10_T10.js (default)
+  Expected no error,
```

Comment:
So I'm _pretty sure_ all the execution tests fail with this error (even without the transpiler on), because the harness code injected by eshost includes
```
      var eshostContext = vm.createContext({
        setTimeout,
        require,
        console,
        print(...args) {
          console.log(...args);
        }
      });
```
and I _don't think_ node 0.10 supported object shorthands/methods/argument spreads.